targets/TARGET_NORDIC/TARGET_NRF5x/TARGET_NRF52/object_owners.c

```diff
@@ -21,10 +21,14 @@
 
 #include <stdio.h>
 
+#if SPI_COUNT
 #define SPI2C_INSTANCES SPI_COUNT
+#else
+#define SPI2C_INSTANCES 0
+#endif
 
-static void * nordic_spi2c_owners[SPI2C_INSTANCES] = { NULL, NULL, NULL };
 
+static void * nordic_spi2c_owners[SPI2C_INSTANCES] = { NULL, NULL, NULL };
```

Comment:
Just a nit, but was this line change intended?

Reply:
No it wasn't, fixing now